code/obj/item/implant.dm

```diff
@@ -316,24 +316,30 @@ THROWING DARTS
 		if(inafterlife(src.owner))
 			return
 		DEBUG_MESSAGE("[src] calling to report crit")
+		SPAWN(rand(15, 25) SECONDS)
 		health_alert()
 		..()
 
 	on_death()
 		if(inafterlife(src.owner))
 			return
 		DEBUG_MESSAGE("[src] calling to report death")
+		SPAWN(rand(15, 25) SECONDS)
 		death_alert()
```

Comment:
Again, indent for anything that's meant to be inside the SPAWN

Reply:
Ditto